shop/shop-controller/src/main/java/shop/woowasap/shop/controller/CartController.java

```diff
@@ -33,4 +37,16 @@ public ResponseEntity<Void> createCartProduct(
         return ResponseEntity.ok().build();
     }
 
+    @PatchMapping
+    public ResponseEntity<Void> updateCartProduct(
+        @RequestBody UpdateCartProductRequest updateCartProductRequest) {
```

Comment:
final을 추가해주셔야 할거같아요~

```suggestion
        @RequestBody final UpdateCartProductRequest updateCartProductRequest) {
```

Reply:
수정했습니당~!